dash/frontend/src/app/modules/private/pages/image/image-scan-result/image-scan-result.component.ts

```diff
@@ -241,11 +241,11 @@ export class ImageScanResultComponent implements OnInit, AfterViewInit, OnDestro
     };
     this.imageService.scanImageModule(this.clusterId, filterImageIds).subscribe(response => {
       if (response.success) {
-        this.alertService.success('Image Scan Queued');
+        this.snackBar.open('Image Scan Queued', 'Close', { duration: 2000 });
         this.setImageScanQueueStatus();
       }
     }, error => {
-      this.alertService.danger(error.error.message);
+      this.snackBar.open(error.error.error.message, 'Close', { duration: 2000 });
```

Comment:
you added an extra 'error' into this access chain compared to what it had before